src/ansys/fluent/core/solver/flobject.py

```diff
@@ -1747,8 +1747,22 @@ def __len__(self):
 
 
 class CreatableNamedObjectMixin(collections.abc.MutableMapping, Generic[ChildTypeT]):
-    """Provides creatable named objects."""
+    """Provides creatable named objects for Fluent 25.1 and later."""
```

Comment:
For our user-facing docs, we are supposed to use this version convention: Fluent 2025 R1.